bindings/java/iota-wallet-java/CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 1.0.0-rc.2 - 2022-11-15
 
+### Added
+
+- Enable native library loading via Java path;
```

Comment:
Is the `NativeApi.ClientCommand` to `WalletCommand` related to that or that's something else ?

Reply:
Its related, I clarified it in the new changelog